examples/create_mesh.py

```diff
@@ -0,0 +1,370 @@
+import subprocess
```

Comment:
I am going to boldly guess that this is a wrong example. cool example though

Reply:
FF - FKK

Reply:
Ready if approved